From f9a89411d68af12acbe72c0dd56e59ed4d3253c0 Mon Sep 17 00:00:00 2001
From: Sinthu Raja <sinthu.raja@ti.com>
Date: Thu, 11 May 2023 13:59:31 +0530
Subject: [PATCH] arm64: dts: ti: k3-am68-sk-rpi*: Override main_gpio0 node

The RPi expansion header main gpio pins are overrided in the overlay by spliting
the GPIO and PWM pins into different nodes. Add main_gpio0 node to override
it correctly with the defined pins

Signed-off-by: Sinthu Raja <sinthu.raja@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am68-sk-rpi-hdr-ehrpwm.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am68-sk-rpi-hdr-ehrpwm.dts b/arch/arm64/boot/dts/ti/k3-am68-sk-rpi-hdr-ehrpwm.dts
index 7f3a20e7443f..b6136fa174a9 100644
--- a/arch/arm64/boot/dts/ti/k3-am68-sk-rpi-hdr-ehrpwm.dts
+++ b/arch/arm64/boot/dts/ti/k3-am68-sk-rpi-hdr-ehrpwm.dts
@@ -63,3 +63,7 @@ &main_ehrpwm4 {
 	pinctrl-0 = <&rpi_header_ehrpwm4_pins_default>;
 	status = "okay";
 };
+&main_gpio0 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&rpi_header_gpio0_pins_default>;
+};
-- 
2.36.1

